### PR TITLE
Add database migration from version 10 to 13

### LINKED ISSUE #968: Migrate from pg10 to pg13
see https://github.com/uyuni-project/uyuni/pull/3795

### PR DESCRIPTION
**Pull request by jordimassaguerpla**
# Description

add database migration from version 10 to 13

# Target branches

Which documentation version does this PR apply to?

- [X] Master (Default)
- [X] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #968

### CODE REVIEW

**Comment by jordimassaguerpla** [comment]
@0rnela See this PR please. I didn't know how to make it understandable, so you might want to rephrase or restructure it. Feel free to edit it.

**Comment by mcalmer** [comment]
Maybe we should say somewhere "SUSE Manager 4.0 run with postgresql 10" and "SUSE Manager 4.1 run with postgresql 12".
When you migrate to SUSE Manager 4.2 which uses postgresql 4.2 please use the appropriate script .....

File: modules/upgrade/pages/db-migration-13.adoc [hunk 1]
```diff
@@ -72,17 +71,20 @@ PostgreSQL stores data at [path]``/var/lib/pgsql/data/``.
 . Start the upgrade:
```
Comment (keichwa) [inline modules/upgrade/pages/db-migration-13.adoc:71]:
```suggestion
. Start the upgrade.
  If you have version 12, run:
```

File: modules/upgrade/pages/db-migration-13.adoc [hunk 2]
```diff
@@ -72,17 +71,20 @@ PostgreSQL stores data at [path]``/var/lib/pgsql/data/``.
...
 +
 ----
+if you have version 12, run:
```
Comment (keichwa) [inline modules/upgrade/pages/db-migration-13.adoc:74]:
```suggestion
```

File: modules/upgrade/pages/db-migration-13.adoc [hunk 3]
```diff
@@ -72,17 +71,20 @@ PostgreSQL stores data at [path]``/var/lib/pgsql/data/``.
...
 /usr/lib/susemanager/bin/pg-migrate-12-to-13.sh
+if you have version 10, run:
```
Comment (keichwa) [inline modules/upgrade/pages/db-migration-13.adoc:76]:
```suggestion
----
+
  If you have version 10, run:
+
----
```

File: modules/upgrade/pages/db-migration-13.adoc [hunk 4]
```diff
@@ -72,17 +71,20 @@ PostgreSQL stores data at [path]``/var/lib/pgsql/data/``.
...
+/usr/lib/susemanager/bin/pg-migrate-10-to-13.sh
 ----
 . When the upgrade has successfully completed, you can safely delete the old database directory and reclaim lost disk space.
-  The old directory is renamed to [path]``/var/lib/pgsql/data-pg12``.
+  The old directory is renamed to [path]``/var/lib/pgsql/data-pg12`` or [path]``/var/lib/pgsql/data-pg10``, depending on whether you had version 12 or 10..
```
Comment (keichwa) [inline modules/upgrade/pages/db-migration-13.adoc:80]:
```suggestion
  The old directory is renamed to [path]``/var/lib/pgsql/data-pg12`` or [path]``/var/lib/pgsql/data-pg10``, depending on the version you started from.
```

File: modules/upgrade/pages/db-migration-13.adoc [hunk 5]
```diff
@@ -101,7 +103,10 @@ If the upgrade fails, the migration script will attempt to restore the database
 . Start the upgrade:
```
Comment (keichwa) [inline modules/upgrade/pages/db-migration-13.adoc:103]:
```suggestion
. Start the upgrade.
  If you are migrating from version 12:
```

File: modules/upgrade/pages/db-migration-13.adoc [hunk 6]
```diff
@@ -101,7 +103,10 @@ If the upgrade fails, the migration script will attempt to restore the database
...
 +
 ----
+If you are migrating from version 12:
```
Comment (keichwa) [inline modules/upgrade/pages/db-migration-13.adoc:106]:
```suggestion
```

File: modules/upgrade/pages/db-migration-13.adoc [hunk 7]
```diff
@@ -101,7 +103,10 @@ If the upgrade fails, the migration script will attempt to restore the database
...
 /usr/lib/susemanager/bin/pg-migrate-12-to-13.sh fast
+If you are migrating from version 10:
```
Comment (keichwa) [inline modules/upgrade/pages/db-migration-13.adoc:108]:
```suggestion
----
+ 
  If you are migrating from version 10:
+
----
```

File: modules/upgrade/pages/db-migration-13.adoc [hunk 8]
```diff
@@ -101,7 +103,10 @@ If the upgrade fails, the migration script will attempt to restore the database
...
+/usr/lib/susemanager/bin/pg-migrate-10-to-13.sh fast
 ----
 . When the upgrade has successfully completed, you can safely delete the old database directory and reclaim lost disk space.
-  The old directory is renamed to [path]``/var/lib/pgsql/data-pg12``.
+  The old directory is renamed to [path]``/var/lib/pgsql/data-pg12`` or [path]``/var/lib/pgsql/data-pg10``, depending on whether you had version 12 or 10..
```
Comment (keichwa) [inline modules/upgrade/pages/db-migration-13.adoc:112]:
```suggestion
  The old directory is renamed to [path]``/var/lib/pgsql/data-pg12`` or [path]``/var/lib/pgsql/data-pg10``, depending on the version you started from..
```

**Review by keichwa — APPROVED** [review]
Thanks you!  This now looks straight forward to me.  My change suggestions hopefully do not look too confusing.  I just tried to separate the explanations ("... run:") from the commands that follow.

**Review by mcalmer — APPROVED** [review]
please apply the suggested changes from Karl. Then it looks good

**Pull request merged**